Web/Edubase.Web.UI/Areas/Groups/Controllers/GroupController.cs

```diff
@@ -548,6 +548,12 @@ public async Task<ActionResult> SaveNewAcademyTrust(CreateAcademyTrustViewModel
 
             if (ModelState.IsValid)
             {
+                //required validation of field GroupId is now only done for groups that are not 16-19
```

Comment:
Had another look at this and think this is probably an ok way to do this.  But maybe this is the wrong place in the method to add the code.

Reply:
FluentValidation would be better but it looks like you've followed a pattern already used in the code and it's probably not the time to be changing all the validation on the page. I think it is in the wrong place though - I'd put it as early in the method as you can

Reply:
Moved the if statement up the method.  As you say we don't have time to restructure it now...